src/pages/commits/ActiveCommits/ChartToggle.tsx

```diff
@@ -42,13 +42,15 @@ export const ChartToggle: React.FC<{
             data-cy="cy-chart-absolute-radio"
             id="chart-radio-absolute"
             value={ChartTypes.Absolute}
+            checked={false}
```

Comment:
We should read the checked variable for both radio buttons from the chartType url query param to make the components controlled